### ISSUE TITLE
Clarification on normalization of data used in Galileo pretraining

### ISSUE THREAD

**Issue by jleben** [nasaharvest/galileo#21]
I could not find in the Galileo paper [1] an unambiguous statement about how the input data was normalized in Galileo model pretraining.

The most firm statement perhaps is found in the README in this repository which suggests to use a provided utility function to prepare data and generate MaskedOutput objects. While the text says "this function optionally normalizes the data against the Galileo pre-training statistics", looking [into the code](https://github.com/nasaharvest/galileo/blob/main/src/data/utils.py#L154), we find `Normalizer(std=False)` which would normalize data to the global min/max rather than the pretraining dataset mean/std.

Could you please clarify what normalization was used to generate the pretrained model weights?

[1] https://arxiv.org/abs/2502.09356

**Comment by gabrieltseng** [comment]
Hi @jleben -

The best source of truth is the training script, https://github.com/nasaharvest/galileo/blob/33ee1dbbd0b38d15e834353f62d8a068b80aa563/train.py#L153 

I apologize for the lack of clarity; I'll update the utils file and add a note in the readme
